Assets/Editor/AssetAuditor/Scripts/AssetAuditorWindow.cs

```diff
@@ -284,12 +293,13 @@ void OnSelectionChange()
 
         private void OnFocus()
         {
-            if (m_Initialized)
+            // this doesn't seem to be needed
+            /*if (m_Initialized)
```

Comment:
This is needed if changes are made to the assets outside of the tools. When refocusing the window it is good to update the UI.

Reply:
I see... Is there a callback that we could use instead? With projects having 1000s of assets, this was really slow (30s+) for rules matching all assets. We could probably optimize something in the loop as well.

Reply:
Ah yeah. Leave it for now I have an idea for something that might work. Ill merge the changes asap.